src/base/bittorrent/session.cpp

```diff
@@ -1136,29 +1139,32 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
     settingsPack.set_int(libt::settings_pack::outgoing_port, outgoingPortsMin());
     settingsPack.set_int(libt::settings_pack::num_outgoing_ports, outgoingPortsMax() - outgoingPortsMin() + 1);
 
-    // Ignore limits on LAN
-    settingsPack.set_bool(libt::settings_pack::ignore_limits_on_local_network, ignoreLimitsOnLAN());
     // Include overhead in transfer limits
     settingsPack.set_bool(libt::settings_pack::rate_limit_ip_overhead, includeOverheadInLimits());
     // IP address to announce to trackers
     settingsPack.set_str(libt::settings_pack::announce_ip, announceIP().toStdString());
     // Super seeding
     settingsPack.set_bool(libt::settings_pack::strict_super_seeding, isSuperSeedingEnabled());
-    // * Max Half-open connections
-    settingsPack.set_int(libt::settings_pack::half_open_limit, maxHalfOpenConnections());
```

Comment:
Libtorrent 1.0 still supports it, isn't it? So we can't remove it. We can hide it in case of libtorrent 1.1.